pennylane/circuit_drawer/circuit_drawer.py

```diff
@@ -55,21 +55,33 @@ def __init__(
         raw_operation_grid,
         raw_observable_grid,
         wires,
-        charset=UnicodeCharSet,
+        charset=None,
         show_all_wires=False,
     ):
         self.operation_grid = Grid(raw_operation_grid)
         self.observable_grid = Grid(raw_observable_grid)
         self.wires = wires
         self.active_wires = self.extract_active_wires(raw_operation_grid, raw_observable_grid)
-        self.charset = charset
+
+        if charset is None:
+            self.charset = CHARSETS["unicode"]()
+        elif isinstance(charset, type) and issubclass(charset, CharSet):
+            self.charset = charset()
+        else:
+            if charset not in CHARSETS:
+                raise ValueError(
+                    "Charset {} is not supported. Supported charsets: {}.".format(
```

Comment:
Makes it slightly clearer to read the error message.
```suggestion
                    "Charset '{}' is not supported. Supported charsets: {}.".format(
```